src/modules/showcomments.js

```diff
@@ -0,0 +1,68 @@
+import commentcounter from './commentscounter.js';
+// const displaydate = () => {
+//   const date = new Date();
+//   const day = date.getDate();
+//   const month = date.getMonth() + 1;
+//   const year = date.getFullYear();
+//   const currentDate = `${day}/${month}/${year}`;
+//   return currentDate;
+// };
```

Comment:
- Kindly, Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. :+1: